src/action_builddir.rs

```diff
@@ -18,18 +17,18 @@ pub fn action_builddir(dir: &Option<PathBuf>, dirs: &RuaDirs, offline: bool, for
 	let dir_str = dir
 		.to_str()
 		.unwrap_or_else(|| panic!("{}:{} Cannot parse CLI target directory", file!(), line!()));
-	wrapped::build_directory(dir_str, &dirs, offline, force);
+	wrapped::build_directory(dir_str, &rua_env.dirs, offline, force);
```

Comment:
I felt that specifying `rua_env.dirs` everywhere makes code easier
to grok, being more explicit. It makes for a noiser diff, though.